ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ExpressionTypeManager.java

```diff
@@ -96,12 +102,26 @@ public ExpressionTypeManager(
   public SqlType getExpressionSqlType(final Expression expression) {
```

Comment:
We can just reuse the method below in this function body

`getExpressionSqlType(expression, Collections.emptyMap(), Collections.emptyList())`